app/src/main/java/org/greenstand/android/TreeTracker/managers/LanguageSwitcher.kt

```diff
@@ -26,7 +29,7 @@ enum class Language(val locale: Locale) {
 class LanguageSwitcher(private val prefs: Preferences) {
 
     fun applyCurrentLanguage(activity: Activity) {
-        val language = Language.fromString(prefs.getString(LANGUAGE_PREF_KEY, "en"))
+        val language = Language.fromString(prefs.getString(LANGUAGE_PREF_KEY, "en") ?: "")
```

Comment:
Why do we need `?: ""` here?  I see the default value of "en" should  take hold if `getString` has null value set. Am I missing to see the need?

Reply:
The default value can be null, so I need to cover that case.